src/components/Home.jsx

```diff
@@ -56,6 +56,7 @@ class Home extends Component {
         <Header>
           <h1>Recipouille</h1>
           <p>Your Trusted Source For Great Food Recipes</p>
+          <Form/>
```

Comment:
@osisamkay when using self-closing elements or components, leave a space before the slash. 
```suggestion
          <Form />
```
Make change and push back for merge